Themis/Models/Assessment.swift

```diff
@@ -5,6 +5,8 @@
 //  Created by Andreas Cselovszky on 14.11.22.
 //
 
+// swiftlint:disable line_length
```

Comment:
I cannot really see a reason to disable line length here. Can't we split the very long lines into multiple ones?